src/coreclr/jit/compiler.cpp

```diff
@@ -4914,6 +4914,8 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
 
         while (iterations > 0)
         {
+            fgModified = false;
```

Comment:
This used to be set to false by `fgComputeDoms` at the end of loop unrolling. It seems better to explicitly do it here, given the comment below on `PHASE_OPT_UPDATE_FLOW_GRAPH`.

(It seems a bit questionable not to run the phase if we loop cloned/unrolled -- I saw some beneficial diffs when I originally didn't have this.)